edx_solutions_api_integration/users/views.py

```diff
@@ -92,6 +93,7 @@
     UserRolesSerializer,
     CourseProgressSerializer,
 )
+from openedx.core.djangoapps.user_api.accounts.image_helpers import get_profile_image_names, get_profile_image_storage
```

Comment:
can you place it along with other openedx imports?

Reply:
Done